docs/faq/env_var.md

```diff
@@ -191,6 +191,10 @@ When USE_PROFILER is enabled in Makefile or CMake, the following environments ca
   - Values: 0, 1 ```(default=1)```
   - Flag to enable or disable MKLDNN accelerator. On by default.
   - Only applies to mxnet that has been compiled with MKLDNN (```pip install mxnet-mkl``` or built from source with ```USE_MKLDNN=1```)
+  
+* MXNET_MKLDNN_CACHE_SIZE
+  - Values: Int ```(default=-1)```
```

Comment:
Why not set to some reasonable default, so that users with variable input shape will not suffer from OOM?

Reply:
It's hard to find a reasonable default for common usage, because it depends on the topology depth. Currently, convolution is defined as stateless op, making all convolution layers share same mkldnn primitive cache pool for reusing purpose. So extremely, for enabling caching for the topology which has 1000 convolution layers with different parameters and data shapes, MXNET_MKLDNN_CACHE_SIZE should be larger than 1000. As time moves, we can imagine that the depth of topology will continue growing.

Reply:
nit: can we be more specific about what the units for cache_size are. Example , number of elements in the cache.